.circleci/rahti-ci.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
```

Comment:
please consider

`setopt -xeuo pipefail`

Reply:
Yes good idea. But should I drop the "x" so we would not print out tokens?

Reply:
hmm, good point. I'm pretty sure circle's _secrets masking_ will take care of that...
https://circleci.com/docs/2.0/env-vars/

Reply:
Yes seems like so, but not completly: "The value of the environment variable is still accessible to users debugging builds with SSH." But this might not be an issue for us, or is it?

Reply:
I think that's fine. To be able to rerun a job with SSH access you need to have write access to the repository on github.